docs/src/workflows/guard.md

```
@@ -1,222 +1,338 @@
-fi
-REPO_PATH="$(cd "$REPO_PATH" && pwd)"
+| Command | Purpose |
+|---|---|
+| `keyhog guard up [--backend <name>]` | Start or ensure the background guard daemon is running and ready. Reconciles registered roots loaded from durable store. |
+| `keyhog guard down` | Stop the background guard daemon cleanly. Persisted root registrations and durable indexes remain on disk. |
+| `keyhog guard add <path> [--mode repo]` | Register a repository or tree for continuous guard protection. Performs initial baseline reconciliation and installs hook before returning. |
+| `keyhog guard list` | Enumerate all registered guard roots, their active states, and terminal sequences. |
+| `keyhog guard status <path> [--format human\|json]` | Print detailed metrics for a guarded root: state, cache hits/misses, files/bytes scanned, residency, and policy digest. |
+| `keyhog guard remove <path>` | Stop guarding a repository and drop its in-memory index and attestation cache to immediately free daemon memory and CPU. |
+| `keyhog guard reconcile <path>` | Force a full baseline reconciliation after intentional policy updates or mass branch operations. |
+| `keyhog guard rebuild <path>` | Delete and recreate the durable guard store for a root after corruption or irrecoverable state. |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)